routing/DELEGATED_CONTENT_ROUTING_HTTP.md

```diff
@@ -118,7 +160,7 @@ limits, allowing every site to query the API for results:
 
 ```plaintext
 Access-Control-Allow-Origin: *
-Access-Control-Allow-Methods: GET, OPTIONS
+Access-Control-Allow-Methods: GET, PUT, OPTIONS
 ```
 
 ## Known Transfer Protocols
```

Comment:
:thought_balloon:  this section grows way faster than the rest of the spec.

Would it be ok to move "Known Transfer Protocols" to a separate file (`KNOWN_TRANSFER_PROTOCOLS.md`)?

Reply:
I recommend doing this in a separate PR to reduce lines changes what are irrelevant to what this PR is trying to introduce.

If this is not a blocker for this PR, I'd be happy to take this as a TODO and get it done once this PR is merged.